experience.tex

```diff
@@ -6,33 +6,35 @@ \section{EXPERIENCE}
 \end{tabular*}
 \begin{bulletlist}
     \item{
-        Worked with the Supply Chain team to determine requirements and metric thresholds for displaying data to
-        the service providers based on their performance for previously completed work items.
+        Assisted building and maintaing greenfield iOS application written in SwiftUI to replace legacy Xamarin application,
+        reducing crash rate from 9\% to < 1\%, reducing time for local development and time to ship a new version of the app,
+        and added thorough logging to the application to make errors actionable for development team.
     }
     \item{
-        Utilized AWS Elasticsearch to improve filtering for assets by properties and to quickly return large lists of
-        matching data to the primary asset web service, lowering asset search response times for the end user.
+        Containerized legacy C\# applications to run in serverless AWS Fargate, moving secrets to HashiCorp Vault to be fetched
+        on container startup, and handled routing traffic securely from Route53, to ALB, to the ECS cluster for a modern
+        deployment and scalability.
     }
     \item{
-        Assisted in translating UI mockups created by the development, sourcing, and UX teams to their final frontend versions using randomly generated placeholder data.
+        Triaged bug reports from users, frequently shipping a new mobile application within an hour of the error being reported.
     }
 \end{bulletlist}
 
 \begin{tabular*}{\textwidth}{l@{\extracolsep{\fill}}r}
-    Software Engineer Intern & September 2018 – May 2019\\
-    Wickidcool Software & Milwaukee, WI\\
+    Consultant & July 2021 – October 2021\\
+    Rebentify Technologies & Milwaukee, WI\\
 \end{tabular*}
 \begin{bulletlist}
     \item{
-        Remotely collaborated with other developers using Git to minimize merge conflicts and implement features
-        simultaneously, improving productivity and reduced development timeline.
+        Set up staging and production environments in GCP Cloud Run and firebase to allow for seamless deployments through integrated
```

Comment:
This is a lot of words - kind of confusingly written. I think the phrasing needs some re ordering...
''''
Set up staging and production environments in GCP Cloud Run and Firebase enabling CI/CD, ensuring environments were differentiated only by security configurations therefore facilitating production-reproducible behavior between tiers. 
''''

Reply:
Also if you are going to capitalize GCP Cloud run then also capitalize Firebase

Reply:
Same for your first comment, concerned with making it too long but I'll see

Reply:
Resolved in a call with you